projboard/tests/test_article.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+from projboard.models.article import Article, Like, View, Subject, User
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+class TestArticle:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_subjects(self):
+        sup = Subject(name='N')
+        sup1 = Subject(name='R')
+        sup.save()
+        sup1.save()
+        return [sup, sup1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_user(self):
+        user = User(name="Anas", nickname="A99", email="Anas@gmail.com", password="Anas123")
+        user1 = User(name="John", nickname="J99", email="John@gmail.com", password="John123")
+        user.save()
+        user1.save()
+        return [user, user1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_articles(self, create_user, create_subjects):
+        articles_list = []
+        for i in range(len(create_user)):
+            article = Article(user_id=create_user[i], title=TITLE,
+                              content=CONTENT, subject_id=create_subjects[i])
+            article.save()
+            articles_list.append(article)
+
+        return articles_list
+
+    # @pytest.mark.skip
+    @pytest.mark.django_db
+    def test_objectCreation(self, create_articles):
+        article = create_articles[0]
```

Comment:
I don't think so, because in this test I'm testing just object creation and not is object saved in db .
I have another test that checks saved in db or not .

Reply:
@Yarboa

Reply:
Is it important to test the creation? 

Thanks for explaining the name, I really did not understand it.
You could skip test_obkectsXXXX methods, it is already covered in migration tests data

Note, you wrote models with methods, 
Tests could help you to understand what is missing in your models,
It also helps to understand how others will call the models you wrote,

Reply:
done